src/main/scala/codacy/swiftlint/SwiftLint.scala

```diff
@@ -76,14 +76,15 @@ object SwiftLint extends Tool {
     config.orElse(nativeConfig)
   }
 
-  private def commandToRun(configOpt: Option[String], files: List[String]): List[String] = {
+  private def commandToRun(configOpt: Option[String], file: String): List[String] = {
     val baseCmd = List("swiftlint", "lint", "--quiet", "--reporter", "json")
 
-    configOpt match {
+    val configCmd = configOpt match {
       case Some(opt) =>
         baseCmd ++ List("--config", opt)
       case None => baseCmd
     }
+    configCmd ++ List(file)
```

Comment:
You can append here:
```suggestion
    configCmd :+ file
```